src/contrib/mlir/runtime/cpu/cpu_callbacks.cpp

```diff
@@ -97,59 +95,58 @@ static bool compare_mkldnn_md_formats(const mkldnn::memory::desc& lhs,
     auto blk2 = md2.format_desc.blocking;
 
     if (blk1.inner_nblks != blk2.inner_nblks ||
-        !compare_mkldnn_dims(blk1.inner_blks, blk2.inner_blks, blk1.inner_nblks) ||
-        !compare_mkldnn_dims(blk1.inner_idxs, blk2.inner_idxs, blk1.inner_nblks))
+        !compareMkldnnDims(blk1.inner_blks, blk2.inner_blks, blk1.inner_nblks) ||
```

Comment:
Another suggestion for future PRs: renaming changes not tightly related to the main goal of this PR could go in a separate PR to have them in quickly and separate from other unrelated changes.

Reply:
Thanks!